skopt/dummy_opt.py

```diff
@@ -52,20 +69,38 @@ def dummy_minimize(func, dimensions, n_calls=1000, random_state=None):
     """
     rng = check_random_state(random_state)
     space = Space(dimensions)
-    X = space.rvs(n_samples=n_calls, random_state=rng)
-
-    init_y = func(X[0])
+    if x0 is None:
+        x0 = []
+    x0 = list(x0)
+    if x0 and not isinstance(x0[0], list):
+        x0 = [x0]
+    n_random_starts = n_calls
+    if y0 is None:
+        y0 = [func(x) for x in x0]
+        n_random_starts -= len(y0)
+    if isinstance(y0, Iterable):
+        y0 = list(y0)
+    else:
+        y0 = [y0]
+    if len(x0) != len(y0):
+        raise ValueError("x0 and y0 should have the same length")
+    X = x0 + space.rvs(n_samples=n_random_starts, random_state=rng)
```

Comment:
How could this possibly work? `n_random_starts` does not exist in `dummy_minimize`